tests/yam/validate/validate_extra_kernel_params.pm

```diff
@@ -0,0 +1,28 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Validate the kernel param of processor.max_cstate=1
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "consoletest";
+use testapi;
+
+sub run {
+    select_console 'root-console';
+    my $output = script_output('cat /proc/cmdline');
+    my $options = get_var('AGAMA_PROFILE_OPTIONS');
+
+    my ($params_str) = $options =~ /bootloader_extra_kernel_params="([^"]*)"/;
+    my @params = split /,/, $params_str;
+
+    foreach my $param (@params) {
+        unless ($output =~ /\Q$param\E/) {
+            die "$param not found in kernel command line!";
+        }
+    }
+    record_info('Extra Kernel Params', "Extra kernel parameters are present: $params_str");
```

Comment:
let's avoid boxes like this, openQA already by design offers too much screenshot, we don't need to remark the result.